packages/docs/src/config.ts

```diff
@@ -48,8 +48,9 @@ export const SIDEBAR = {
 
     { text: 'Page Exports', header: true },
     { text: 'About Page Exports', link: 'en/page-exports/about' },
-    { text: 'Basic', link: 'en/page-exports/basic' },
-    { text: 'Relay', link: 'en/page-exports/relay' },
+    { text: 'Page Exports - Basic', link: 'en/page-exports/basic' },
+    { text: 'Page Exports - Advanced', link: 'en/page-exports/advanced' },
+    { text: 'Page Exports - Relay', link: 'en/page-exports/relay' },
```

Comment:
Do we need this "Page Exports - " at front of the title? It looks a bit duplicated since we already have a heading that indicates the page exports section.

Reply:
oh my bad! I didn't realise this was for the menu, thought it was the page titles/routes. I need to look more at how astro works haha

Reply:
fixed!